docs/en/source/gitGuideline.rst

```diff
@@ -0,0 +1,46 @@
+********************************
+Branch Management Guidelines 
+********************************
+
+.. figure:: figures/gitGuideline.jpg
+
+Community can have two main branches with an infinite lifetime:
+
+1. **Master branch**:This is a highly stable branch that is always production-ready and contains the last release version of source code in production.
+2. **Development branch**: Derived from the master branch, the development branch serves as a branch for integrating different features planned for an upcoming release. This branch may or may not be as stable as the master branch. It is where developers collaborate and merge feature branches.All of the changes should be merged back into master somehow and then tagged with a release number.
+
+
+Apart from those two primary branches, there are other branches in the workflow:
+
+- **Feature Branch**:Forked from the development branch for bugs fixing and feature development. Merged back to the development branch after bug fixing and feature development.
+
+- **Hotfix branch**:Hotfix branches are created from the master branch. It is the current production release running live and causing troubles due to a severe bug.But changes on develop are yet unstable. We may then branch off a hotfix branch and start fixing the problem. It should be rarest occasion, in case only critical bugs.
+
+**Note**:Only NLE and NECTI members have privilege to create and merge Hotfix branch.
+
+.. list-table::  **Branch naming convention** 
+   :widths: 20 40 40
+   :header-rows: 1
+
+   * - Branch				
+     - Branches naming guideline	 		 
+     - Remarks
+     
+   * - Feature branches
+     - Must branch from: *develop*. Must merge back into: *develop* .Branch naming convention: *feature-feature_id*						
+     - *feature_id* is the Github issue id from **https://github.com/ScorpioBroker/ScorpioBroker/issues**
+
+   * - Bug Branches
+     - Must branch from: *develop*. Must merge back into: *develop* .Branch naming convention: *bug-bug number*						
+     - *Issueid* is the Github issue id from **https://github.com/ScorpioBroker/ScorpioBroker/issues**
+
+   * - Hotfix Branches
+     - Must branch from: *master branch*.Must merge back into: *master branch*.Branch naming convention: *hotfix-bugnumber*.						
+     - *Bugnumber* is the Github issue id from **https://github.com/ScorpioBroker/ScorpioBroker/issues**
+
+Permissions to the branches:
+*******************************
+
+- **Master** - We tend to very strict that only NLE members and Naveen/Ajay can merge on Master branch and accept the pull requests. Pull requests to master can be raised by only NECTI OR NLE members. 
+
+- **Development** - Any community member can raise the pull request to the develop branch but it should be review by NLE or NECTI members.
```

Comment:
one more line should be added below last line.

Development branches commits will be moved to master branch only when all the test cases written under NGSI-LD test suites, will run successfully.